libs/hwes/eventstoredb/entity_event.go

```diff
@@ -0,0 +1,63 @@
+package eventstoredb
+
+import (
+	"context"
+	"fmt"
+	"telemetry"
+
+	"github.com/EventStore/EventStore-Client-Go/v4/esdb"
+	"github.com/google/uuid"
+	"google.golang.org/protobuf/proto"
+	"hwes"
+)
+
+type EntityAggregate struct {
```

Comment:
what is an entity aggregate? how does it differ from a normal aggregate? why are they needed?